unstructured_ingest/v2/pipeline/steps/chunk.py

```diff
@@ -77,8 +73,8 @@ async def _run_async(
         return ChunkStepResponse(file_data_path=file_data_path, path=str(output_filepath))
 
     def get_hash(self, extras: Optional[list[str]]) -> str:
-        hashable_string = json.dumps(
-            self.process.config.to_dict(), sort_keys=True, ensure_ascii=True
+        hashable_string = serialize_base_model_json(
```

Comment:
This will hash with plaintext secrets. I can't remember is someone unhashes, will they see the secrets?

Reply:
This should be safe.